micro-rdk/src/common/conn/server.rs

```diff
@@ -411,19 +426,14 @@ where
             }
         }
     }
-    async fn serve_http2<U>(
-        &self,
-        mut c: U,
+    async fn serve_http2(
```

Comment:
This function is now simple enough it could probably just be inlined above, but I think I'd like to keep it rather than making `serve` more complex.

Reply:
agreed